db/controllers/users.controller.js

```diff
@@ -0,0 +1,10 @@
+const app = require("../app")
+const {fetchUsers} = require("../models/users.model")
+
+
+exports.selectAllUsers = (request, response, next) => {
+    fetchUsers()
+    .then((retrievedUsers) => {
+        return response.status(200).send({users: retrievedUsers})
+    })
```

Comment:
Please add a .catch() block here to catch any errors and to let your error handling middlewares to deal with them.